docs/doc/12-unload-data/index.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Unloading Data from Databend
+slug: ./
+---
+
+Databend provides a variety of tools for loading and unloading data. One of the methods of unloading data from Databend is using the [`COPY INTO <location>`](../14-sql-commands/10-dml/dml-copy-into-location.md) command. 
```

Comment:
Users would want to know what the other ways to unload are if we say "One of the methods of unloading data...". Could you shed some light?

Reply:
Like using MySQL client to query all the data into a file, but we don't recommend this way.